client/context/client_manager.go

```diff
@@ -0,0 +1,46 @@
+package context
```

Comment:
This is interesting, but with only round-robin load balancing it's not that useful in my opinion. I would rather have a discussion about what we want this to do for clients - maybe track failed requests, or track latency, and automatically switch clients to provide the best UX. I'm not convinced we need this right now (but if you think we do by all means explain why).

Reply:
The round-robin load balancing is indeedly simple. But it does contribute to TPS, please refer to the my test [report](https://github.com/HaoyangLiu/cosmos-sdk/blob/haoyang/lcd-test/docs/light/Cosmos-LCD-Test-Report.md). I think a load balancing module is necessary. Maybe currently we can just remove this module. But in long run, we have to add load balancing. How do you think about this?